docs/types-of-interviews.md

```diff
@@ -11,7 +11,7 @@ This section contains the most common types of interviews you'll come across whe
 Behavioral questions attempt to gauge how you would behave in stressful or otherwise adverse conditions. There are a couple common types of behavioral questions:
 
 - **"Tell me about a time..."** These questions are based on the idea that the best predictor of future behavior is past performance. The goal is to get you to present real examples of how you have handled things like conflict and failure in the past so the company can have some insight into how you may behave in the future.
-- **Scenarios.** Rather than looking for past examples, scenario question present you with situations and you're asked how you would handle them. Interviewers are generally looking for problem solving and empathy skills when asking these questions.
+- **Scenarios.** Rather than looking for past examples, scenario questions present you with situations and you're asked how you would handle them. Interviewers are generally looking for problem solving and empathy skills when asking these questions.
```

Comment:
I am making the minimal fix here.  However, the following may flow better:
```suggestion
- **Scenarios.** Scenario questions provide situations and ask how to handle them instead of examining past examples. Interviewers are generally evaluating your problem-solving and empathy skills when asking these questions.
```